torch/onnx/_internal/jit_utils.py

```diff
@@ -332,3 +376,11 @@ def parse_node_kind(kind: str) -> Tuple[str, str]:
     if "::" in opname:
         raise ValueError(f"Node kind: {kind} is invalid. '::' should only apear once.")
     return domain, opname
+
+
+@_beartype.beartype
+def is_custom_domain(domain: str) -> bool:
+    """Check if the domain is official."""
+    return domain not in {"onnx", "aten", "prim"} and not domain.startswith(
+        "org.pytorch"  # to include ATen
```

Comment:
@justinchuby @BowenBao this function is not robust, as I found that there could by `org.pytorch` family staying in the onnx graph, and they look like official domain somehow. Not sure if I miss other "family here". Could you guys confirm if this logic is sufficient?

Reply:
I would separate domains in torchscript and onnx in the implementation. Is `org.pytorch` a torch concept or a onnx concept? If it is a onnx concept, I would handle it in a different place

Reply:
TODO: move it to onnx_utils.py

Reply:
Suggest to change

Reply:
Removed